catalog/spatial/wfs/spatial-wfs-mapper/pom.xml

```diff
@@ -35,6 +35,35 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-collections4</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.fastjson</groupId>
+            <artifactId>boon</artifactId>
+            <version>0.30</version>
```

Comment:
https://github.com/codice/ddf/blob/master/pom.xml#L152

Reply:
changed to refer to property